cmd/get/task.go

```diff
@@ -38,15 +39,24 @@ Retrieves particular version of task by name within project and domain.
 
 ::
 
- flytectl get workflow -p flytesnacks -d development  core.basic.lp.greet --version v2
+ flytectl get task -p flytesnacks -d development  core.basic.lp.greet --version v2
 
-Retrieves project by filters.
+Retrieves all the tasks with filters.
 ::
-
- Not yet implemented
+  
+  bin/flytectl get task -p flytesnacks -d development --filter.field-selector="task.name=k8s_spark.pyspark_pi.print_every_time,task.version=v1" 
+ 
+Retrieve a specific task with filters.
```

Comment:
Same here, elaborate specifc here since you are already passing the name of the task.